go/vt/tabletmanager/agent.go

```diff
@@ -247,10 +250,21 @@ func (agent *ActionAgent) Tablet() *topo.TabletInfo {
 }
 
 // Healthy reads the result of the latest healthcheck, protected by mutex.
+// If that status is too old, it means healthcheck hasn't run for a while,
+// and is probably stuck, this is not good, we're not healthy.
 func (agent *ActionAgent) Healthy() (time.Duration, error) {
 	agent.mutex.Lock()
 	defer agent.mutex.Unlock()
-	return agent._replicationDelay, agent._healthy
+
+	healthy := agent._healthy
+	if healthy == nil {
+		timeSinceLastCheck := time.Now().Sub(agent._healthyTime)
```

Comment:
could we use UTC time instead ? I have some past experience that a local time may be messed up by daylight saving. In this case, it may cause "last health check is too old".